examples/graphene_apps/python_worker/README.md

```diff
@@ -61,7 +61,7 @@ https://creativecommons.org/licenses/by/4.0/
 - Before building and running application for Graphene-SGX, we need to install Intel SGX driver and Graphene SGX driver.
 
   - To install Intel SGX driver please refer https://github.com/hyperledger/avalon/blob/master/PREREQUISITES.md#intel-sgx-in-hardware-mode
-  - To install Graphene SGX driver please refer https://graphene.readthedocs.io/en/latest/building.html#install-the-graphene-sgx-driver-not-for-production
+  - Clone the Graphene codebase from [here](https://github.com/oscarlab/graphene.git) if not done already. Then, to install Graphene SGX driver please refer https://graphene.readthedocs.io/en/latest/building.html#install-the-graphene-sgx-driver-not-for-production. Also, note that you might have to install a few other dependencies if you have not built Graphene on this setup. Refer [this](https://graphene.readthedocs.io/en/latest/building.html#required-packages) for assistance.
```

Comment:
We are not going to build graphene stand alone. GSC tool will download and build it. Did you face any issue if you didn't install the dependencies ?

Reply:
I already had Graphene built earlier on my setup. I might have to do a fresh install to verify it. But, since the driver installation step in the documentation of Graphene mentioned `make`, I suspect the dependencies might throw up issues there.

Reply:
we can mention install GSC tool dependent software packages from https://github.com/oscarlab/graphene/blob/master/Documentation/manpages/gsc.rst#software-packages